src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -1537,6 +1537,79 @@ bool ONNXConcatOp::inferShapes() {
   return true;
 }
 
+//===----------------------------------------------------------------------===//
+// Split
+
+bool ONNXSplitOp::inferShapes() {
+  if (!getOperand().getType().cast<RankedTensorType>()) {
+    emitError("Input tensor not ranked");
+    return false;
+  }
+
+  int numOfResults = getNumResults();
+  auto inputType = getOperand().getType().cast<RankedTensorType>();
+  auto inputShape = inputType.getShape();
+  auto inputRank = inputShape.size();
+
+  // Checking value of axis parameter.
+  auto axisIndex = axis().getSExtValue();
+  // Negative axis means values are counted from the opposite side.
+  if (axisIndex < 0) {
+    axisIndex = inputRank + axisIndex;
+    auto builder = mlir::Builder(getContext());
+    axisAttr(builder.getI64IntegerAttr(axisIndex));
+  }
+  if (axisIndex >= inputRank) {
+    emitError("Split axis value out of bound");
+    return false;
+  }
+
+  // Checking value of split parameter.
+  auto splitOpt = split();
```

Comment:
`splitOpt` -> `splitAttr`.

Reply:
I used `splitAttribute` since `splitAttr` is a function to update the split attribute.